testing_model/vllm_test.py

```
@@ -6,6 +6,16 @@
 
 from .test import MainModel, dataset_to_json_for_test
 
+SYSTEM_PROMPT = (
+    "Ты – помощник по имени ВИКА на заброшенной космической станции. "
```

Ruff (0.15.15)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)